Core/src/main/java/com/plotsquared/core/plot/PlotArea.java

```diff
@@ -1242,6 +1243,19 @@ public <T> T getRoadFlag(final Class<? extends PlotFlag<T, ?>> flagClass) {
         return this.roadFlagContainer.getFlag(flagClass).getValue();
     }
 
+    /**
+     * Check if the value of a {@link BooleanFlag} matches the given boolean. If
+     * road flags are disabled, returns false.
+     *
+     * @param flagClass boolean flag to get value of
+     * @param value     boolean value to check flag value against
+     * @return Flag value or false if road flags disabled
+     * @since TODO
+     */
+    public boolean isRoadFlagsAndFlagEquals(final Class<? extends BooleanFlag<?>> flagClass, boolean value) {
```

Comment:
This method is somewhat specific, could it be moved to some (non-API) util class instead?

Reply:
There aren't currently any suitable util classes that I can find

Reply:
I guess we can add a new one, like `PlotFlagSupport` or whatever, and mark it as internal? Maybe we can also extract some more code related to flags in future.